### ISSUE TITLE
Cannot install FormEncode 1.3.1

### ISSUE THREAD

**Issue by gotcha** [buildout/buildout#556]
`FileNotFoundError` due to `os.rename` into a missing directory.

@jensens  found it via https://github.com/plone/buildout.coredev/runs/2057051639

**Comment by jugmac00** [comment]
I tried to reproduce it and it worked on my machine.

Can you tell me what I did wrong?

```
❯ bat buildout.cfg 
───────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: buildout.cfg
───────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ [buildout]
   2   │ parts = one
   3   │ 
   4   │ [one]
   5   │ recipe = zc.recipe.egg
   6   │ eggs = formencode
   7   │ 
   8   │ [versions]
   9   │ formencode = 1.2.1
───────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

/tmp/buildout-test via 🐍 v3.6.9 (venv)
❯ buildout
Installing one.
Getting distribution for 'formencode==1.2.1'.
Got FormEncode 1.2.1.
```

**Comment by gotcha** [comment]
@jugmac00 `FormEncode 1.3.1` 😉

**Comment by jugmac00** [comment]
Oh my 😔

Thank you!

**Comment by gotcha** [comment]
Fixed in #557
